web/themes/custom/pineapple/sass/layouts/nodes/_recipe-page.scss

```diff
@@ -0,0 +1,84 @@
+.row {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
The row class should be removed from this file and added to the base file as it can be reusable in other places.

Reply:
done